locale/en/blog/release/v8.0.0.md

```diff
@@ -0,0 +1,1216 @@
+application to crash. Starting with 8.0.0 such errors will be silently ignored,
+making use of `console.log()` and the other APIs safer. It will be possible to
+maintain legacy behavior related to errors via an `ignoreErrors` option passed
+to the `Console` constructor.
+
+### Static Error Codes
+
+We have started the process of assigning static error codes to all errors
+generated by Node.js. While it will take some time for every error to be
+assigned a code, a handful have been updated within 8.0.0. These error codes
+are guaranteed not to change even if the error type or message changes.
+
+Codes are manifest to the user in two ways:
+
+* Using the `code` property on `Error` object instances
+* Printing the `[ERR_CODE]` in the stack trace of an Error
+
+For instance, calling `assert(false)` will generate the following
+`AssertionError`:
+
+```js
+> assert(false)
+AssertionError [ERR_ASSERTION]: false == true
+    at repl:1:1
+    at ContextifyScript.Script.runInThisContext (vm.js:44:33)
+    at REPLServer.defaultEval (repl.js:239:29)
+    at bound (domain.js:301:14)
+    at REPLServer.runBound [as eval] (domain.js:314:12)
+    at REPLServer.onLine (repl.js:433:10)
+    at emitOne (events.js:120:20)
+    at REPLServer.emit (events.js:210:7)
+    at REPLServer.Interface._onLine (readline.js:278:10)
+    at REPLServer.Interface._line (readline.js:625:8)
+```
+
+Information for static error codes can be queried quickly by referencing the
+Node.js documentation. For instance, the URL to lookup information about the
+`ERR_ASSERTION` error code is
+https://nodejs.org/dist/latest-v7.x/docs/api/errors.html#ERR_ASSERTION.
+
+### Redirecting Process Warnings
+
+Processing warnings such as deprecations may now to redirected to a file by
+using either the `--redirect-warnings={file}` command line argument or matching
+`NODE_REDIRECT_WARNINGS={file}` environment variable. Rather than printing
+warnings out to `stderr` by default, warnings will be written out to the
+specified file, allowing those to be analyzed separately from an application's
+primary output.
+
+### Stream API improvements
+
+For users of the Stream API, new standard mechanisms for destroying and
+finalizing Stream instances have been added. Every `Stream` instance will now
+inherit a `destroy()` method, the implementation of which can be customized and
+extended by providing a custom implementation of the `_destroy()` method.
+
+### Debugger changes
+
+The legacy command line debugger is being removed in Node.js 8. The underlying
+debugging protocol has been deprecated in V8 for several years now, and will
+finally be removed. Therefore, Node.js is removing it completely. As a command
+line replacement, `node-inspect` has been integrated directly into the Node.js
+runtime. Additionally, the V8 Inspector debugger, which arrived previously as
+an experimental feature in Node.js 6, is being upgraded to an fully supported
+feature.
```

Comment:
`an fully supported` -> `a fully supported`

I think this paragraph is a fair bit too wordy. Something succinct might be better. Maybe drop a couple sentences so it looks like this?:

> The legacy command line debugger is being removed in Node.js 8. As a command
> line replacement, `node-inspect` has been integrated directly into the Node.js
> runtime. Additionally, the V8 Inspector debugger, which arrived previously as
> an experimental feature in Node.js 6, is being upgraded to a fully supported
> feature.